packages/esm-appointments-app/src/home-appointments/links.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Link } from '@carbon/react';
+import { Add, Calendar } from '@carbon/react/icons';
+
+import { useConfig, navigate } from '@openmrs/esm-framework';
+import PatientSearch from '../patient-search/patient-search.component';
+import { launchOverlay } from '../hooks/useOverlay';
+import { spaBasePath } from '../constants';
+
+import styles from './appointments-list.scss';
+
+const SeeAllAppointmentsLink = () => {
+  const { useBahmniAppointmentsUI: useBahmniUI } = useConfig();
+
+  const { t } = useTranslation();
+
+  return useBahmniUI ? (
+    <Link
+      size="md"
+      target="_blank"
+      className={'cds--btn cds--btn--ghost ' + styles.seeAllLink}
```

Comment:
The Carbon `Button` component takes a `href` prop. Instead of transforming a link to look like a button wouldn't it make more sense to use the Button component directly?

Reply:
oh I didn't know that, I will then use a button.